pkg/objectstore/objectstore.go

```diff
@@ -13,7 +13,9 @@ type ObjectStore interface {
 
 	// DeleteBucket removes a bucket from the object store.
 	DeleteBucket(string) error
+}
 
-	// GetRegion returns the region for a given bucket
+type ObjectStoreWithRegion interface {
```

Comment:
Since (for now) we only need this in case of amazon, this contract could be owned by our amazon object store wrapper. (lacking a better name: the one in the internal directory 😄 ) It could also be unexported, since it's only used there.